pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -429,6 +429,9 @@ func (r *Builder) mapResources(vm *model.Workload, object *cnv.VirtualMachineSpe
 			object.Template.Spec.Domain.CPU.IsolateEmulatorThread = true
 		}
 	}
+	if object.Instancetype != nil {
```

Comment:
The `DedicatedCPUPlacement` and `IsolateEmulatorThread` above also collide with instancetypes.